<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>261</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>241</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Choose value of step:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>41</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>x = </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>90</y>
       <width>41</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>y = </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="x_edit">
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>61</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="y_edit">
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>100</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="solve_button">
     <property name="geometry">
      <rect>
       <x>158</x>
       <y>180</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Solve</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Error_edit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.05</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Error = </string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>311</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Swis721 Hv BT</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Gauss-Seidel Method</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>621</width>
      <height>501</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>10</y>
       <width>331</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Choose value of temperature:</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>72</y>
       <width>551</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>173</y>
       <width>551</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>373</y>
       <width>551</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>272</y>
       <width>551</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>80</y>
       <width>20</width>
       <height>401</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>79</y>
       <width>20</width>
       <height>401</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_7">
     <property name="geometry">
      <rect>
       <x>449</x>
       <y>79</y>
       <width>20</width>
       <height>401</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_8">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>80</y>
       <width>20</width>
       <height>401</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_9">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>80</y>
       <width>20</width>
       <height>401</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_10">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>472</y>
       <width>551</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="T12_edit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>121</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>25</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>204</x>
       <y>110</y>
       <width>41</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>T = </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="T13_edit">
     <property name="geometry">
      <rect>
       <x>386</x>
       <y>121</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>25</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>110</y>
       <width>41</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>T = </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="T24_edit">
     <property name="geometry">
      <rect>
       <x>526</x>
       <y>221</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>210</y>
       <width>41</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>T = </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="T34_edit">
     <property name="geometry">
      <rect>
       <x>526</x>
       <y>321</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>310</y>
       <width>41</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>T = </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="T21_edit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>221</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>200</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>64</x>
       <y>210</y>
       <width>41</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>T = </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="T31_edit">
     <property name="geometry">
      <rect>
       <x>109</x>
       <y>321</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>200</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>63</x>
       <y>310</y>
       <width>41</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>T = </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="T42_edit">
     <property name="geometry">
      <rect>
       <x>249</x>
       <y>421</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>75</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>203</x>
       <y>410</y>
       <width>41</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>T = </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="T43_edit">
     <property name="geometry">
      <rect>
       <x>386</x>
       <y>421</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>75</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>410</y>
       <width>41</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>T = </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>203</x>
       <y>210</y>
       <width>41</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>T = </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>343</x>
       <y>210</y>
       <width>41</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>T = </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>203</x>
       <y>310</y>
       <width>41</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>T = </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>342</x>
       <y>310</y>
       <width>41</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Swis721 Hv BT</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>T = </string>
     </property>
    </widget>
    <widget class="QLabel" name="T22_edit">
     <property name="geometry">
      <rect>
       <x>247</x>
       <y>220</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QLabel" name="T23_edit">
     <property name="geometry">
      <rect>
       <x>389</x>
       <y>220</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QLabel" name="T32_edit">
     <property name="geometry">
      <rect>
       <x>249</x>
       <y>320</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QLabel" name="T33_edit">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>321</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>988</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
